react-interactive/src/LayoutGrid.tsx

```diff
@@ -21,7 +21,7 @@
 //
 //******************************************************************************************************
 
-import React from 'react'
+import * as React from 'react'
 
 interface IProps {
     rowsPerPage: number
```

Comment:
```suggestion
    RowsPerPage: number
```

Reply:
While, we are already in here making some changes. We might as well capitalize the props in here to be consistent with other components in gemstone. Also, it wouldn't be a bad idea to add annotated comments to the props as well since they are going to be exported.

Reply:
If you need example of any of the annotated comments, all of the components in gpa-gemstone/react-forms should have them.